.github/workflows/integration.yaml

```diff
@@ -86,9 +61,14 @@ jobs:
       if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.debug_enabled != 'false' }}
       timeout-minutes: 15
 
-    - name: Delete and Regenerate Lockfile
-      if: ${{ matrix.lockfile == 'no-lockfile' }}
-      run: rm yarn.lock && CI="" yarn install #currently broken
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '14'
```

Comment:
16 is LTS now

Reply:
And we can probably turn that cache off, its really cool and really fast but technically we aren't using it for anything